components/Primitives/src/SwitchPresenter/SwitchConverter.cs

```diff
@@ -66,13 +66,19 @@ public SwitchConverter()
         SwitchCases = new CaseCollection();
     }
 
+    /// <summary>
+    /// Converts the provided value based on the defined cases.
+    /// </summary>
```

Comment:
Could this be inheritdoc tag as it's from the IValueConverter interface? (same with below?)